### PR TITLE
Support DataFrame uploads as Parquet in Python client

### PR DESCRIPTION
**Pull request by neumark**

### CODE REVIEW

File: examples/clients/python/seafowl/dataframe.py [hunk 1]
```diff
@@ -63,7 +70,23 @@ def make_row_inserter(conn: SeafowlConnectionParams) -> RowInserter:
...
+def export_to_parquet_and_upload(
+    data: pd.DataFrame,
+    conn: SeafowlConnectionParams,
+    destination_table: QualifiedTableName,
+) -> None:
+    url = f"{get_db_endpoint(conn.url, conn.database)}/upload/{destination_table.schema or DEFAULT_SCHEMA}/{destination_table.table}"
+    with tempfile.NamedTemporaryFile(suffix=".parquet") as tmpfile:
+        data.to_parquet(tmpfile, compression="gzip")
+        tmpfile.flush()
+        tmpfile.seek(0)
```
Comment (gruuya) [inline examples/clients/python/seafowl/dataframe.py:82]: Just out of curiosity, is this `seek` needed? Will Seafowl re-use the same file handle?

**Review by gruuya — APPROVED** [review]

File: examples/clients/python/seafowl/dataframe.py (hunk 1 above)
Comment (mildbyte) [reply examples/clients/python/seafowl/dataframe.py:82]: The `requests.post` uses this `tmpfile`, maybe it doesn't rewind automatically.

**Review by mildbyte — APPROVED** [review]

File: examples/clients/python/seafowl/dataframe.py (hunk 1 above)
Comment (neumark) [reply examples/clients/python/seafowl/dataframe.py:82]: Right, I don't think it rewinds automatically.

Comment (gruuya) [reply examples/clients/python/seafowl/dataframe.py:82]: Understood, thanks!

**Pull request merged**